README.md

```diff
@@ -44,23 +54,23 @@ This Integration Daemon postgres connector supports the following types of datab
 
 # Starting the Egeria Platform
 Below is an example of the java commandline used to launch the Egeria platform that will host the Integration Daemon that is in turn hosting the 
-postgres connector instance we are about to configure.
-The point of interest when launching a platform that will host connectors is the `-Dloader.path=/home/wbittles/egeria-database-connectors/postgres-connector/libs` command line option.
-The option specifies a file system directory that must contain both the postgres JDBC driver and the postgres-connector jar file.
+Postgres connector instance we are about to configure.
+The point of interest when launching a platform that will host connectors is the `-Dloader.path=/home/wbittles/egeria-database-connectors/Postgres-connector/libs` command line option.
+The option specifies a file system directory that must contain both the Postgres JDBC driver and the Postgres-connector jar file.
```

Comment:
can we include the actual file here (and no capitalization)

Reply:
I think the example to get the postgres connector is super. Postgres itself should be reasonably obvious so I think this is enough